wpilibc/src/main/native/include/frc/smartdashboard/Mechanism2d.h

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#pragma once
+
+#include <map>
+
+#include "networktables/NetworkTableEntry.h"
+#include "frc/smartdashboard/Sendable.h"
+#include "frc/smartdashboard/SendableHelper.h"
+#include "frc/util/Color8Bit.h"
+
+#include "MechanismRoot2d.h"
+
+namespace frc {
+
+class Mechanism2d : public Sendable, public SendableHelper<Mechanism2d> {
+  const wpi::Twine& kBackgroundColor{"backgroundColor"};
```

Comment:
Twine is only meant to be a temporary object, so this will not work. See https://llvm.org/docs/ProgrammersManual.html#llvm-adt-twine-h.

Replace with `static constexpr const char*`.